src/tests/test_utils/common_test_utils/include/common_test_utils/file_utils.hpp

```diff
@@ -365,7 +365,7 @@ class MockPlugin : public ov::IPlugin {
 }  // namespace ov
 
 
-// vpu repo uses CommonTestUtils::
+// npu repo uses CommonTestUtils::
 // so we need to add these names to CommonTestUtils namespace
 namespace CommonTestUtils {
```

Comment:
@olpipi  Why this code is still here??

Reply:
https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/5374